src/Components/List/ContactsList.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { SafeAreaView, SectionList } from 'react-native';
+import EveryContact from './EveryContact';
+import SectionTitle from './SectionTitle';
+
+const name = [
```

Comment:
please move all this constant data into constant directory, and move generateData, compare and changeDataStructure functions into utils